index.html

```diff
@@ -7,21 +7,27 @@
     <link rel="stylesheet" href="styles.css">
     <title>Kiplaacollo Portfolio website</title>
   </head>
-  <body>
+  <body>        
+    <header>
+      <nav class="navbar">
+        <a href="#" class="logo head">Welcome</a>
```

Comment:
Your image            |  Original image
:-------------------------:|:-------------------------:
<img width="340" alt="Screenshot 2023-01-03 at 18 29 21" src="https://user-images.githubusercontent.com/69011963/210411655-45514337-ca55-42c3-a39d-9df708f61271.gif"> | <img src="https://user-images.githubusercontent.com/69011963/210411611-ec6225b3-4bfd-4d12-88cc-f4205924f969.png" width="1440"/>

- [ ] It appears that the scrolling width and hamburger button placement are not in line with the original design. Please adjust the web app's width so that it remains constant, and ensure that the hamburger button is in its intended location.